packages/browser-repl/src/components/editor.tsx

```diff
@@ -3,12 +3,13 @@ import AceEditor from 'react-ace';
 import { Autocompleter } from '@mongosh/browser-runtime-core';
 import { AceAutocompleterAdapter } from './ace-autocompleter-adapter';
 
-import 'brace/ext/language_tools';
-import 'brace/mode/javascript';
+import 'ace-builds/src-noconflict/ext-language_tools';
+import 'ace-builds/src-noconflict/mode-javascript';
 import './ace-theme';
 
-import ace from 'brace';
-const tools = ace.acequire('ace/ext/language_tools');
+import ace from 'ace-builds';
+// TODO: can we not just import this?
+const tools = ace.require('ace/ext/language_tools');
```

Comment:
To be determined. No idea why this did ace.acequire before? Can we not just `import tools from 'ace-builds/src-noconflict/ext-language_tools';` above? And if so why didn't we always do that?